server/app/models/volume.rb

```diff
@@ -39,4 +39,13 @@ def services
     self.grid.grid_services.where("service_volumes.volume_id" => self.id)
   end
 
+  def driver_for_node(host_node)
```

Comment:
Missing specs (and docs) ... also wondering if we really need this middleman method here.